@xen-orchestra/lite/src/components/host/dashboard/HostDashboardCpuUsageChart.vue

```diff
@@ -59,7 +59,7 @@ const cpuUsage = computed<LinearChartData>(() => {
 })
 
 const maxValue = computed(() => {
-  const values = cpuUsage.value[0]?.data.map(item => item.value ?? 0) ?? []
+  const values = cpuUsage.value[0]?.data.map(item => Number(item.value) ?? 0) ?? []
```

Comment:
Number(undefined) return NaN. but NaN ?? 0 = NaN

```suggestion
  const values = cpuUsage.value[0]?.data.map(item => item.value ? Number(item.value) : 0) ?? []
```

Or same as `HostDashboardLoadAverageChart.vue`

```suggestion
  const values = cpuUsage.value[0]?.data.map(item => Number(item.value) || 0) ?? []
```

warning, same on all file